.changeset/selfish-books-fail.md

```diff
@@ -0,0 +1,10 @@
+---
+"@comet/admin-date-time": major
+"@comet/admin": major
+---
+
+All `FinalFormField` components now support the `required` prop
+
+The `clearable` prop is removed from `FinalFormDatePicker`
+
+The clear button in `FinalFormDatePicker` is visible for all non-`required` fields
```

Comment:
Please split into two changesets, one for `@comet/admin` and one for `@comet/admin-date-time`. While the changes are related, they're still separate changes IMO.